coremltools/converters/mil/mil/ops/defs/tensor_transformation.py

```diff
@@ -753,25 +769,32 @@ def value_inference(self):
 @register_op(doc_str="")
 class transpose(Operation):
     """
-    Permutes tensor ``x`` dimensions according to ``perm``.
-
+    Permute tensor ``x`` dimensions according to ``perm``.
+    
     Parameters
     ----------
     x: tensor<*?, T> (Required)
-        * Must be at least 1D. ``x`` may have symbolic shape.
+        * Must be at least 1-D. ``x`` may have a symbolic shape.
     perm: const<[rank(x)], i32> (Required)
         * Permutation order. Must be non-negative integers.
-
+    
     Returns
     -------
     tensor<*?,T>
         * Tensor with same rank and type as ``x``.
-
+    
     Attributes
     ----------
     T: fp32
+    
+    References
+    ----------
+        * `TF <https://www.tensorflow.org/api_docs/python/tf/transpose?version=nightly>`_
```

Comment:
one reference is enough, say torch.Tensor.permute

Reply:
OK